modules/imgproc/src/color.cpp

```diff
@@ -7910,8 +7944,8 @@ void cvtBGRtoYUV(const uchar * src_data, size_t src_step,
     }
 #endif
 
-    static const float yuv_f[] = { 0.114f, 0.587f, 0.299f, 0.492f, 0.877f };
-    static const int yuv_i[] = { B2Y, G2Y, R2Y, 8061, 14369 };
+    static const float yuv_f[] = { R2YF, G2YF, B2YF, R2VF, B2UF };
+    static const int yuv_i[] = { R2Y, G2Y, B2Y, R2VI, B2UI };
     const float* coeffs_f = isCbCr ? 0 : yuv_f;
     const int* coeffs_i = isCbCr ? 0 : yuv_i;
```

Comment:
Wouldn't the result be wrong in case `isCbCr` is true? It looks like `RGB2YCrCb_i` use a bit different conversion coefficients in such case, however it still stores `Cb` channel to last byte of the triplet.